OmniSharp.sln.GhostDoc.xml

```diff
@@ -0,0 +1,53 @@
+<GhostDoc>
```

Comment:
is this file needed?

Reply:
Nope, was just an extension I had installed.